utils/release-candidate-testing/python/requirements.txt

```diff
@@ -0,0 +1 @@
+valkey-glide
```

Comment:
define a version

Reply:
Nope. It's just for the test that it works when working locally, we don't use it in CI (we use the tag) and it should always bring the latest.
So no reason to define.